src/site/development-assets/gulpfile.js

```diff
@@ -0,0 +1,39 @@
+var gulp = require('gulp');
```

Comment:
Are these all built in or require installs? If they require installed you must create a `dependencies.css` with each of the remote packages listed and then update the depinstall.sh file to run those installs. You must assume that the server running the code has nothing installed

Reply:
They require installs, but it's only for development, and even so it's not necessary, only convenient. It's there in case someone decides they want to recompile the SCSS using gulp. There's no need to create a bunch of node dependencies, and if it's avoidable then it's better this way. If someone decides to use gulp then they'll know how to deal with this.

Reply:
If a different contributor wanted to edit the website would he/she require gulp?

Reply:
No. It's purely for convenience. If you really want, I'll initialize a node project in the repository with gulp and subsidiary packages as dev-dependencies. But it seems like overkill.

Reply:
If it isn't necessary I'd still create the dependencies.node file and add to the depinstall.sh but just comment out the dependencies in the dependencies.node so that if someone wanted to they could just uncomment it and it would run.